### ISSUE TITLE
Gemini issue

### ISSUE THREAD

**Issue by marinsokol5** [kroitor/ccxt#30]
Latest version of ccxt used and python 3.6.

When calling private post method on gemini.
This pops out:
```    
 File "C:\oop\novi ws\Crypto\TransactionFees.py", line 120, in get_transaction_fee
    response = self.request(command, type='private', method='POST')
  File "C:\Program Files (x86)\Python36-32\lib\site-packages\ccxt\__init__.py", line 6557, in request
    payload = base64.b64encode (self.json (request))
  File "C:\Program Files (x86)\Python36-32\lib\base64.py", line 58, in b64encode
    encoded = binascii.b2a_base64(s, newline=False)
TypeError: a bytes-like object is required, not 'str'

```

**Comment by kroitor** [comment]
Fixing this very soon.

**Comment by kroitor** [comment]
Can you please test the fix and report whether the problem is gone or not?

**Comment by marinsokol5** [comment]
Yea the error is gone.

**Comment by kroitor** [comment]
Thanks!

**Comment by LevinTheDream** [comment]
Found this bug again with gemini exchange when requesting balance.

**Comment by kroitor** [comment]
Working on it! (and the BCC issue as well) Thanks! Both will be fixed today, hopefully.

**Comment by kroitor** [comment]
Oops, it was issue 113, not 112 )

@LevinTheDream I changed it slightly, can you please try it again (ccxt v1.3.85)? I don't have an option for getting the keys from Gemini, they're still verifying us. If you can send us a zero-balance keypair, we could then test its private API on our own and fix it quicker. Otherwise we're will have to ask you to test our fixes.

**Comment by LevinTheDream** [comment]
Great! Just tested with v1.3.86, and everything is working as expected.
Thanks!

@kroitor Sorry don't have a zero-balance keypair, maybe you can use gemini-sandbox for testing? I don't think they require verification.

**Comment by kroitor** [comment]
@LevinTheDream ok, thank you very much for your feedback!

**Comment by kroitor** [comment]
@LevinTheDream I'm sorry can I ask for one more thing? Can you please try registering with gemini sandbox? We tried it (earlier) and we don't get email confirmations from them, though they say, that it has been sent to us. We wanted to find out if their sandbox site is working at all or not (may be an email-problem on our side).

**Comment by LevinTheDream** [comment]
Yes I signed up with them a while ago, and received confirmation the same day.

**Comment by kroitor** [comment]
@LevinTheDream thx! We will try harder then )

**Comment by cjpetrus** [comment]
is this problem still occuring in py 3.6?

**Comment by kroitor** [comment]
@cjpetrus it shouldn't reappear, however, if you still see it reoccurring, let us know, we will investigate further.

**Comment by kroitor** [comment]
@cjpetrus ok, thanks, will look into it and get back to you.

**Comment by kroitor** [comment]
@cjpetrus what's your version of ccxt?

**Comment by kroitor** [comment]
@cjpetrus fixed in 1.10.353. Thx!
